framework/src/reporters/ParsedVectorRealReductionReporter.C

```diff
@@ -9,22 +9,24 @@
 
 #include "ParsedVectorRealReductionReporter.h"
 
-registerMooseObject("OptimizationApp", ParsedVectorRealReductionReporter);
+registerMooseObject("MooseApp", ParsedVectorRealReductionReporter);
 
 InputParameters
 ParsedVectorRealReductionReporter::validParams()
 {
   InputParameters params = ParsedReporterBase::validParams();
   params.addClassDescription(
       "Use a parsed function to iterate through a vector and reduce it to a scalar.");
-  params.addRequiredParam<ReporterName>("reporter_name", "Reporter name with vector to reduce.");
+  params.addRequiredParam<ReporterName>("vector_reporter_name",
+                                        "Reporter name with vector to reduce.");
   params.addRequiredParam<Real>("initial_value", "Value to intialize the reduction with.");
 
   // reporter_symbols are the two symbols for reduction value and current value for the reduction
   // operation, these symbols are enforced in the constructor with a mooseError
   params.set<std::vector<std::string>>("reporter_symbols") = {"reduction_value", "indexed_value"};
   params.suppressParameter<std::vector<std::string>>("reporter_symbols");
-
+  params.set<std::vector<std::string>>("scalar_reporter_symbols") = {};
+  params.suppressParameter<std::vector<std::string>>("scalar_reporter_symbols");
   // This reporter is for postprocessing optimization results and should be exectuted at the end of
```

Comment:
I think TIMESTEP_END is already the default for UOs so you can delete this comment and the line below